src/pages/Vault/VaultItem/index.tsx

```diff
@@ -132,6 +137,10 @@ export const VaultItemUi: React.FC<VaultItemUiProps> = ({
           {readableUserStakedTokens}
         </Typography>
 
+        {xvs && areTokensEqual(stakedToken, xvs) && (
```

Comment:
Now that we can get the index of the pool relevant to Prime from the Prime contract, I think we should use this to define whether the `PrimeStatusBanner` component should be rendered (you can use `useGetPrimeStatus`).
We should also only render it if the Prime feature is enabled.